public_html/wp-content/plugins/wc-post-types/wc-post-types.php

```diff
@@ -859,7 +859,7 @@ public function add_slides_info_to_session_posts( $content ) {
 			return $content;
 		}
 
-		$session_slides_html  = '<div class="session-video">';
+		$session_slides_html  = '<div class="session-slides">';
```

Comment:
🤔 , changing this breaks backwards compatibility. A few options:

1) Do it anyway, since using `video` for the slides was a bug.
2) Add additional `slides` class to the slides link, leaving the `video` class. Organizers could use something like `:not( '.session-slides' )` to target the `video` class only.
3) Add additional classes to both links, e.g., `session-slides-container` and `session-video-container`. Then add HTML comments letting organizers know the old ones are deprecated.

I don't think `#2` is good, since it's unintuitive, and extra work for organizers. `#1` would require manually reviewing the CSS for all past camps, and updating if necessary, or letting those styles break. So, I think `#3` is the best option.